modules/update-preparing-evaluate-apirequestcount-workloads.adoc

```diff
@@ -25,22 +25,20 @@ For example:
 +
 [source,terminal]
 ----
-$ oc get apirequestcounts flowschemas.v1beta1.flowcontrol.apiserver.k8s.io -o yaml
+$ oc get apirequestcounts csistoragecapacities.v1beta1.storage.k8s.io -o yaml
 ----
 +
 You can also use `-o jsonpath` to extract the `username` and `userAgent` values from an `APIRequestCount` resource:
 +
 [source,terminal]
 ----
-$ oc get apirequestcounts flowschemas.v1beta1.flowcontrol.apiserver.k8s.io \
+$ oc get apirequestcounts csistoragecapacities.v1beta1.storage.k8s.io \
   -o jsonpath='{range .status.currentHour..byUser[*]}{..byVerb[*].verb}{","}{.username}{","}{.userAgent}{"\n"}{end}' \
   | sort -k 2 -t, -u | column -t -s, -NVERBS,USERNAME,USERAGENT
 ----
 +
 .Example output
 [source,terminal]
 ----
-VERBS  USERNAME                                                            USERAGENT
-get    system:serviceaccount:openshift-cluster-version:default             cluster-version-operator/v0.0.0
-watch  system:serviceaccount:openshift-oauth-apiserver:oauth-apiserver-sa  oauth-apiserver/v0.0.0
+TODO: need example output
```

Comment:
@sanchezl Same questions here, on whether the above commands are right (again, couldn't test since I didn't have that in my cluster). And then I'd also need some help with what the example output here would look like for the above command, if you can please help with that. Thanks!